packages/mermaid/src/diagrams/git/gitGraphRenderer.js

```diff
@@ -87,11 +110,31 @@ const drawCommits = (svg, commits, modifyGraph) => {
   const sortedKeys = keys.sort((a, b) => {
     return commits[a].seq - commits[b].seq;
   });
+
+  const isParallelCommits = gitGraphConfig.parallelCommits;
+  const layoutOffset = 10;
+  const commitStep = 40;
   sortedKeys.forEach((key) => {
     const commit = commits[key];
+    const posWithOffset = pos + layoutOffset;
+
+    if (isParallelCommits) {
+      if (!commit.parents.length) {
+        pos = 0;
+        if (dir === 'TB') {
+          pos = 30;
+        }
+      } else {
+        const closestParent = findClosestParent(commit.parents);
+        pos =
+          dir === 'TB'
+            ? commitPos[closestParent].y + commitStep
+            : commitPos[closestParent].x + commitStep;
+      }
+    }
```

Comment:
Shouldn't posWithOffset be set after the if block, as pos is updated inside it?

Reply:
Yes it should, it even breaks the tests I added, not sure how I missed this. Good catch.